bundle-linux.sh

```diff
@@ -101,3 +101,5 @@ else
   mkdir -p "$out/$bin_dir"
   cp "$binary" "$out/$bin_dir"
 fi
+
+find $out -empty -type d -delete
```

Comment:
This could be done in default.nix and cover both platforms, right?

Reply:
yes, and done.